EIPS/eip-2612.md

```diff
@@ -0,0 +1,204 @@
+---
+eip: 2612
+title: permit: 712-signed approvals
```

Comment:
I *think* the bot isn't happy with the `:` here.  Try this and see if it satisfies the bot:
```suggestion
title: permit - 712-signed approvals
```